careers/technical-engineer.md

```diff
@@ -16,63 +16,67 @@ The senior engineer displays a mastery of the core competencies of software engi
 
 **Collaboration:** They ensure that they do not become a single point of failure for their domain by mentoring and training others. When relevant, they work effectively with others outside the organization (whether in other departments or with vendors). They are capable of effectively running projects of any size independently.
 
+### Responsibilities
+
+* Provide technical leadership and guidance across the product and engineering organizations.
+* Make key architectural decisions and ensure the scalability, performance, and reliability of critical systems.
+* Mentor and coach junior and mid-level engineers to foster their technical growth.
+* Collaborate with product managers, designers, and stakeholders to define requirements and solutions.
```

Comment:
> Partner with product managers, designers, and stakeholders to validate requirements, identifying potential issues early and guiding teams toward viable technical solutions.

Reply:
I feel like your proposed change actually dials back the expectations here. To me, it sounds more passive: they're not helping define requirements, they just validating them; they're not defining the solution, they're just guiding the team toward good solutions.

At senior, I'd expect them to have a heavier hand in owning requirements and implementation.

Thoughts?

Reply:
What do you think about something closer to this?

> Partner with product managers, designers, and stakeholders to define and own requirements. Identify technical risks early and lead the team in architecting solutions that support business objectives.